static/css/pad.css

```diff
@@ -57,6 +57,7 @@ textarea {
 
 /* Position the online count button a bit better */
 li[data-key=showusers] > a #online_count {
+    background-color: #000;
```

Comment:
I think Etherpad (not our plugin) creates the markup for the online count, so our plugin doesn't directly create the HTML. In the code where we hook the event for a change in the online count, we could remove the markup that Etherpad inserts and place our own markup within the button, but that seems pretty clunky.